plugins/com.google.cloud.tools.eclipse.appengine.whitelist/plugin.xml

```diff
@@ -4,7 +4,7 @@
 
   <extension
       id="appEngineWhitelist"
-      name="App Engine JRE Whitelist"
+      name="%extension.name"
```

Comment:
Does this work? I see `Bundle-Localization: plugin` in `MANIFEST.MF` in many bundles, but not in this bundle. I may be wrong about this though.